bgpd/bgp_vty.c

```diff
@@ -73,8 +73,8 @@
 #endif
 
 FRR_CFG_DEFAULT_BOOL(BGP_IMPORT_CHECK,
-	{ .val_bool = true, .match_profile = "datacenter", },
-	{ .val_bool = false },
+	{ .val_bool = false, .match_version = "< 7.4", },
```

Comment:
This is incorrect and will break loading `7.3 datacenter` configs.  You need 
`{ .val_bool = false, .match_version = "< 7.4", .match_profile = "traditional", }`